hikari/errors.py

```diff
@@ -317,7 +318,8 @@ def __str__(self) -> str:
             try:
                 value += _dump_errors(self.errors).strip("\n")
             except KeyError:
-                value += data_binding.dump_json(self.errors, indent=2)
+                # Use the stdlib json.dumps here to be able to indent
+                value += json.dumps(self.errors, indent=2)
```

Comment:
Could a function like `data_binding.json_pretty_print` work here?

Reply:
Since we are only doing this in this one specific place rn, its fine for this to be a one off